lib/BackgroundJobs/IndexerJob.php

```diff
@@ -117,15 +117,19 @@ protected function index(array $files): void {
 			if (!$file instanceof File) {
 				continue;
 			}
-			try {
-				$fileHandle = $file->fopen('r');
-			} catch (LockedException|NotPermittedException $e) {
-				$this->logger->error('Could not open file ' . $file->getPath() . ' for reading', ['exception' => $e]);
-				continue;
-			}
 			$userIds = $this->storageService->getUsersForFileId($queueFile->getFileId());
 			foreach ($userIds as $userId) {
 				try {
+					try {
+						$fileHandle = $file->fopen('r');
+					} catch (LockedException|NotPermittedException $e) {
+						$this->logger->error('Could not open file ' . $file->getPath() . ' for reading', ['exception' => $e]);
+						continue 2;
```

Comment:
maybe it is just a not permitted exception, we should just continue the userId loop I guess

Reply:
this keeps the logic the same, see old code: we aborted processing the file before the userId loop if there was an exception

Reply:
sure, but this can be an improvement over that. Also maybe not the best time to suggest this but should we re-queue the locked files?

Reply:
```suggestion
						continue;
```

alright :)